### PR TITLE
Branch01

### PR DESCRIPTION
**Pull request by ercancivi**

### CODE REVIEW

**Review by Olesea1986 — APPROVED** [review]
Ladies are approving, good job guys!!!!

File: src/test/java/com/trycloud/tests/UserStrories/US_2_Modules_verifications/US_2_Test.java [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package com.trycloud.tests.UserStrories.US_2_Modules_verifications;
 
 import com.trycloud.tests.base.HomePage;
+import com.trycloud.utilities.ConfigurationReader;
```
Comment (altyshaibraim) [inline src/test/java/com/trycloud/tests/UserStrories/US_2_Modules_verifications/US_2_Test.java:4]: i see

**Review by altyshaibraim — APPROVED** [review]
thanks

**Review by IrinaMakaryan — APPROVED** [review]
nice job!

**Pull request merged**